common/script/content.coffee

```diff
@@ -55,6 +55,24 @@ api.mystery =
   wondercon: {start:'2014-03-24',end:'2014-04-01'} # not really, but the mechanic works
 _.each api.mystery, (v,k)->v.key = k
 
+equipmentList =
+  'weapon': {localeKey: 'weapon', isEquipment: true, needsLowerCase: false}
+  'armor' : {localeKey: 'armor', isEquipment: true, needsLowerCase: false}
+  'head' : {localeKey: 'headgear', isEquipment: true, needsLowerCase: false}
+  'shield' : {localeKey: 'offhand', isEquipment: true, needsLowerCase: false}
+  'back' : {localeKey: 'back', isEquipment: true, needsLowerCase: false}
+  'body' : {localeKey: 'body', isEquipment: true, needsLowerCase: false}
+  'headAccessory' : {localeKey: 'headAccessory', isEquipment: true, needsLowerCase: false}
+  'eyewear' : {localeKey: 'eyewear', isEquipment: true, needsLowerCase: false}
+  'hatchingPotions' : {localeKey: 'hatchingPotion', isEquipment: false, needsLowerCase: false}
+  'eggs' : {localeKey: 'eggSingular', isEquipment: false, needsLowerCase: false}
+  'quests' : {localeKey: 'quest', isEquipment: false, needsLowerCase: false}
+  'Saddle' : {localeKey: 'foodSaddleText', isEquipment: false, needsLowerCase: true}
+  get : (key, w = window) ->
```

Comment:
This function won't be needed. As I said, we just need to create a new string for the lowercase version of "saddle". We'll call `window.env.t` on the function that actually uses the `localeKey`.